src/react_native/core.cljs

```diff
@@ -94,11 +94,12 @@
 
 (defn use-effect
   ([effect-fn]
+   (use-effect effect-fn []))
+  ([effect-fn deps]
    (react/useEffect
     #(let [ret (effect-fn)]
-       (if (fn? ret) ret js/undefined))))
-  ([effect-fn deps]
-   (react/useEffect effect-fn (bean/->js deps))))
+       (if (fn? ret) ret js/undefined))
+    (bean/->js deps))))
```

Comment:
This fixes a crash on `use-effect`, I think it was only fixed for the case of no dependencies passes through, so modified to work on both scenarios

Reply:
Previously I was using `effect!` from `quo.react-native`, and did this change in order to refactor to use `use-effect` from `react-native.core`